Classes/Domain/Model/Product/Tea.php

```diff
@@ -60,4 +65,20 @@ public function setImage(FileReference $image): void
     {
         $this->image = $image;
     }
+
+    /**
+     * @return int<0, max>
```

Comment:
Do we need the annotations? I'm asking because other methods like `getTitle()` don't have them any more and I always prefer to have as less maintenance as possible.

Reply:
In general, we don't have the PHPDoc type annotations if they do not add anything over the native type declaration (neither more precise type information nor a relevant human-readable description). In this case, we make the type more precise, which in turn helps PHPStan detect incorrect calls: We change it from the native type declaration `int` to the more specific `int<0, max>` (i.e., a non-negative int).

I'm using `@phpstan-var` instead of `@var` for the property as having this type annotation confused Extbase.

(I do not insist on having these more precise type annotations. I'd prefer them, but if they block the +1, I can remove them.)

Reply:
Thanks for the info. Maybe that's something we should document somewhere if not already done. I remember there was a discussion about that regarding TYPO3 core source code as well? Are you involved in that topic? Maybe that's nothing to document for EXT:tea but https://docs.typo3.org/m/typo3/reference-coreapi/12.4/en-us/CodingGuidelines/CglPhp/CodingBestPractices/Index.html?

Reply:
Yes, we should document this there.

I'm also going to add a comment here in our code explaining why I'm using `@phpstan-var` here.

Reply:
Done.